week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,25 @@
+/*
+
+*I feel this code all looks very hacky.
+
+*/
+
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+var array = [mentor1, mentor2, mentor3, mentor4, mentor5]
```

Comment:
I'm not sure `array` is the best name for an array. Can you think of a more meaningful one?

Reply:
Ditto on the trying to self-learn before starting this course. Thanks for pointing this out and it has been revised.